Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3
+FROM alpine@sha256:2171658620155679240babee0a7714f6509fae66898db422ad803b951257db78
```

Comment:
Do you specifically want to lock to this version? The worry is that we would miss security patches that make it into Alpine. I'd stick with alpine:3 (or alpine:3.something) to make sure we automatically get them included

Reply:
Fixed to v3